### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#583] [Galaxy Trivia] [answered]
🟩⬇️🟨🟥
🟥⬛⭐⚙️
🟥🟨👑🟥

**Comment by peterbaccaro** [comment, marked as answer]
1) BENIN
2) ANGOLA
3) SPAIN

![flags2](https://user-images.githubusercontent.com/74666079/232843210-1dacd922-3c38-4b91-9346-1f284950280b.png)

**Comment by alejandroch1202** [comment]
- Benin 🇧🇯
- Angola 🇦🇴
- España 🇪🇸

**Comment by Veercodeprog** [comment]
Brazil
ANGOLA
SPAIN

**Comment by thuongtruong1009** [comment]
The first flag seems to be green, black, and red, which is commonly associated with a few countries, but the most well-known is Egypt. The second flag is red with a star and some gears, which is the flag of China. The last flag is red, yellow, and black with a crown in the middle, which is the flag of Belgium. Therefore, my guess is that the country is Belgium.
